src/config.js

```diff
@@ -16,7 +16,7 @@ const FALSY_INPUT = new Set(["disable", "false", "none", "no", "0"]);
 export const AUTH_KEY = process.env.AUTH_KEY;
 if (!AUTH_KEY && !process.env.JEST_WORKER_ID) {
     throw new Error(
-        "AUTH_KEY env variable is required, it is not possible to authenticate requests without it",
+        "AUTH_KEY env variable is required, it is not possible to authenticate requests without it"
```

Comment:
Don't you apply a formatted since the start? Why would it change the result suddenly?

Reply:
I had set prettier requirements to `2.8` and up, and at some point my local version was `3.0+` which changed the default value of the `[trailing-comma](https://prettier.io/docs/en/options.html#trailing-commas)` rule from `es5` to `all`. I moved back to enforcing 2.8 (for consistency with Odoo's codebase) and forgot some files.